extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/memo.stack3/Paste State.pushbutton/script.py

```diff
@@ -321,13 +318,13 @@ def set_tansform_matrix(selvp, selview):
                             else:
                                 raise OriginalIsViewDrafting
                     except IOError:
-                        forms.alert('Could not find saved viewport '
+                        errors.append((vport, 'Could not find saved viewport '
```

Comment:
As now it iterates through many Viewports, first collect all errors. Do not show an Alert on each viewport